docs/_documentations/che-setupregistries.md

```diff
@@ -1,17 +1,21 @@
 ---
 layout: docs
-title: Adding an image registry in Codewind in Che
-description: Adding an image registry in Codewind in Che
+title: Adding image registries in Codewind for Che
+description: AAdding image registries in Codewind for Che
```

Comment:
typo ? AAdding  ?

Reply:
Good spot! Acrolinx checker does not check these tags it seems...Cheers Mark, all fixed in most recent PR.